corehq/ex-submodules/dimagi/utils/django/email.py

```diff
@@ -26,6 +27,29 @@
 def send_HTML_email(subject, recipient, html_content, text_content=None,
                     cc=None, email_from=settings.DEFAULT_FROM_EMAIL,
                     file_attachments=None, bcc=None):
+    if settings.SERVER_ENVIRONMENT in ['production', 'staging']:
+        _send_HTML_email_punt.delay(subject, recipient, html_content, text_content=text_content,
+            cc=cc, email_from=email_from,
+            file_attachments=file_attachments, bcc=bcc)
+    else:
+        _send_HTML_email_run(subject, recipient, html_content, text_content=text_content,
+            cc=cc, email_from=email_from,
+            file_attachments=file_attachments, bcc=bcc)
+
+
+@task(queue="email_queue_punt",
+      bind=True, default_retry_delay=15 * 60, max_retries=10, acks_late=True)
+def _send_HTML_email_punt(subject, recipient, html_content, text_content=None,
```

Comment:
When you have `bind=True` I think tha the first argument should be `self`

Reply:
updated